docs/runbooks/KubeMacPoolDuplicateMacsFound.md

```diff
@@ -0,0 +1,31 @@
+# KubeMacPoolDuplicateMacsFound
+
+## Meaning
+
+KubeMacPool is responsible to allocate mac addresses and avoid mac address conflicts.
+Upon starting KubeMacPool, it scan the current VM instances and updates it's database.
+In case it find duplicates it increase a counter of mac duplicate, which results in this alert.
+Duplicates will not be updated in the database but do exists until solved.
```

Comment:
This is not accurate, I would vote to remove this line.. we should keep the explanation generic and simple
```suggestion
```